### DISCUSSION TITLE
替换字体后图片匹配不上

### DISCUSSION THREAD

**Discussion by cyatar** [NateScarlet/auto-derby#215] [Q&A] [answered]
育成时显示  no match: tmpl=('single_mode_character_detail_button.png',)
经检查发现 名称为 single_mode_character_detail_button.png 的图片和 我的电脑育成时对应位置不匹配（文字位置对应不上） 那么是否直接替换图片即可正常运行？

**Comment by NateScarlet** [comment, marked as answer]
至少需要 debug/last_screenshot.png 才能确定是什么问题

[问题反馈](https://github.com/NateScarlet/auto-derby#%E9%97%AE%E9%A2%98%E5%8F%8D%E9%A6%88)

推荐阅读[《提问的智慧》](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)

**Reply by cyatar** [reply]
![last_screenshot](https://user-images.githubusercontent.com/94457811/145576540-4aac192b-4a17-4020-928f-d390d6310e04.png)
[launcher.log](https://github.com/NateScarlet/auto-derby/files/7692849/launcher.log)
[auto_derby.log](https://github.com/NateScarlet/auto-derby/files/7692850/auto_derby.log)

抱歉没有阅读完问题反馈中的内容

**Reply by NateScarlet** [reply]
不支持替换字体 Duplicate of #16 #209

**Reply by cyatar** [reply]
好的 谢谢
